contracts/drop/DropERC721.sol

```diff
@@ -247,11 +247,13 @@ contract DropERC721 is
         baseURI[baseURIIndex] = _baseURIForTokens;
         baseURIIndices.push(baseURIIndex);
 
-        if (_encryptedBaseURI.length != 0) {
-            encryptedBaseURI[baseURIIndex] = _encryptedBaseURI;
+        (bytes memory encryptedURI, bytes32 provenanceHash) = abi.decode(_data, (bytes, bytes32));
```

Comment:
@nkrishang what do I pass here for regular minting? I tried passing 2 empty strings packed with the correct types and it fails here

Reply:
this is what i'm passing
```
 const data = ethers.utils.solidityPack(
      ["bytes", "bytes32"],
      [ethers.utils.toUtf8Bytes(""), ethers.utils.formatBytes32String("")],
    );
    ```